src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -93,63 +104,75 @@ Vec3f D_80B8AA04[] = {
     { 470.0f, 30.0f, 140.0f },   { 410.0f, 80.0f, 130.0f },   { 410.0f, 80.0f, 130.0f },   { 0.0f, 0.0f, 0.0f },
 };
 
-Color_RGBA8 D_80B8AB24[] = {
-    { 255, 255, 90, 255 },
-    { 55, 25, 80, 255 },
-    { 5, 195, 110, 255 },
+typedef enum KaizokuLipstickColors {
+    /* 0x0 */ KAIZOKU_LIPSTICK_YELLOW,
+    /* 0x1 */ KAIZOKU_LIPSTICK_PURPLE,
+    /* 0x2 */ KAIZOKU_LIPSTICK_SEAGREEN, // (between aqua and green)
+} KaizokuLipstickColors;
+
+Color_RGBA8 sKaizokuLipstickColors[] = {
+    { 255, 255, 90, 255 }, // KAIZOKU_LIPSTICK_YELLOW
+    { 55, 25, 80, 255 },   // KAIZOKU_LIPSTICK_PURPLE
+    { 5, 195, 110, 255 },  // KAIZOKU_LIPSTICK_SEAGREEN
 };
 
-Color_RGBA8 D_80B8AB30[] = {
-    { 255, 130, 10, 255 },
-    { 185, 130, 210, 255 },
-    { 135, 195, 80, 255 },
+typedef enum KaizokuOutfitColors {
+    /* 0x0 */ KAIZOKU_OUTFIT_ORANGE,
+    /* 0x1 */ KAIZOKU_OUTFIT_LAVENDER,
+    /* 0x2 */ KAIZOKU_OUTFIT_GREEN,
+} KaizokuOutfitColors;
+
+static Color_RGBA8 sKaizokuOutfitColors[] = {
+    { 255, 130, 10, 255 },  // KAIZOKU_OUTFIT_ORANGE
+    { 185, 130, 210, 255 }, // KAIZOKU_OUTFIT_LAVENDER
+    { 135, 195, 80, 255 },  // KAIZOKU_OUTFIT_GREEN
 };
 
-typedef enum EnKaizokuDamageEffect {
+typedef enum KaizokuDamageEffect {
     /* 0x0 */ KAIZOKU_DMGEFF_NONE,
-    /* 0x1 */ KAIZOKU_DMGEFF_1,
-    /* 0x2 */ KAIZOKU_DMGEFF_2,
+    /* 0x1 */ KAIZOKU_DMGEFF_STUNNED,
```

Comment:
```suggestion
    /* 0x1 */ KAIZOKU_DMGEFF_STUN,
```
I would prefer this for a few reasons:
1. It's more consistent with other enemy actors
2. It differentiates it a little more from `KAIZOKU_DMGEFF_STUNNED_ONLY`